roles/load_aware/load_aware_deploy_trimaran/tasks/main.yml

```diff
@@ -102,6 +102,10 @@
     delay: 5
     retries: 20
     until: trimaran_test_pod_state.stdout == 'Completed'
+  rescue:
+    - name: Ensure error is raised
+      fail:
+        msg: "{{ ansible_failed_result }}"
```

Comment:
you don't need this, the error is automatically raised
it's like in python:
```
try:
   ...
finally:
   ... # always executed
```
and what you've written is:
```
try:
   ...
except:
   raise
finally:
   ... # always executed
```

Reply:
wait, I didn't know about `ansible_failed_result`. What good does it bring? 🤔

Reply:
`ansible_failed_result` should just store the result of the failed block. I was running into an issue where a task in the block was failing silently and caused a bit of a headache. I added this and made a few tweaks to make sure that to make sure that it failed. It might have been one of the other tweaks that fixed it though, so if it looks wrong I'll change it back and give it a shot.

Reply:
hum, would you be able to find the link to the silent failure? 

I'm _almost_ certain that this block shouldn't help, but it doesn't do anything wrong either, so keep it for now